frontend/vue/components/CodeExercise/CodeExercise.vue

```diff
@@ -12,12 +12,22 @@
       <ExerciseActionsBar
         class="code-exercise__editor-block__actions-bar"
         :is-running="isKernelBusy"
+        :is-api-token-needed="isApiTokenNeeded"
         :run-enabled="isKernelReady"
         :grade-enabled="isKernelReady && isGradingExercise"
         @run="run"
         @grade="grade"
       />
     </div>
+    <div v-if="isApiTokenNeeded" class="code-exercise__api-token-info">
+      <WarningIcon />
+      <div>
+        This code is using an IBMQ provider. If you want to execute it, you need to setup your API-token in
+        <BasicLink url="/account">
```

Comment:
@korgan00 @techtolentino ideally once time we have decided how are we going to let the URL's for the profile page we should update this redirect to go to the specific page.